src/Sylius/Bundle/ApiBundle/Resources/config/serialization/Commands/Account/RequestResetPasswordToken.xml

```diff
@@ -19,7 +19,7 @@
         <attribute name="email">
             <group>shop:reset_password:create</group>
         </attribute>
-        <attribute name="localeCode" serialized-name="locale">
+        <attribute name="localeCode">
```

Comment:
Checking just in case, shouldn't this group change be reflected in some api test?